src/Core/SettingsChangesHistory.cpp

```diff
@@ -713,8 +713,7 @@ const VersionToSettingsChangesMap & getMergeTreeSettingsChangesHistory()
         addSettingsChanges(merge_tree_settings_changes_history, "25.4",
         {
             /// Release closed. Please use 25.5
-            {"refresh_parts_interval", 0, 0, "A new setting"},
```

Comment:
This has been removed in https://github.com/ClickHouse/ClickHouse/pull/79018, hope that there will be no more conflicts